test/migrations.unit.spec.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
+
+import fullSchema1010ez from '../src/js/hca/config/form';
```

Comment:
I'm a little concerned that we have more than one place that devs are expected to update when they add a new form. It's hard to surface these places effectively.

Reply:
Great point @jbalboni, I'll add a test to check for that, similar to what @cvalarida designed for the user-profile helpers test.